### ISSUE TITLE
Does not run as unprivileged user

### ISSUE THREAD

**Issue by marcogaio** [AceCentre/FaceCommander#75]
I've just installed version  0.9.3 on a Win10 64bit box.

As privileged user (eg, in 'Administrators' group) work as expected; for an unprivileged user, does not run, and don't emit errors or so; simply does not run.

Thanks.

**Comment by willwade** [comment]
Ok. @alex5105 i think like the work you are doing on portable installer would fix this. 

@marcogaio there is a problem - and this would exist on any app that wants to take over mouse or keyboard actions. You can’t control full screen uiaccess apps (eg Grid3 Tobii Dynavox software ) unless they are installed as admin with the uiaccess flag. But if you don’t need that we should be able to make a “portable “ app that runs wherever you put it

**Comment by marcogaio** [comment]
But I have installed as admin, and then used as unprivileged user... i'm a sysadmin and i use scripted provisioning to install software on users pc (currently, with a tool called WPKG).

I've installed software with commandline:
`"%SOFTWARE%\WPKG\FaceCommander-Installer-v0.9.3.exe" /sp- /verysilent /suppressmsgboxes /norestart /lang="it" /mergetasks="desktopicon"`

from administrators user (precisely: SYSTEM use), and then i've tried to run it from an unprivileged one.

**Comment by willwade** [comment]
Ok. I think this is all the same issue actually. It’s to do with a) the config and logging paths the app uses and b) the installer needing admin rights and c) the app needing to be signed / with uiaccess flag 

@marcogaio nothing you can do until we fix a couple of things

**Comment by willwade** [comment]
@marcogaio try 0.9.4 (latest release)

**Comment by marcogaio** [comment]
> @marcogaio try 0.9.4 (latest release)

No, still does not work: app start, open the debug window, then crash.

Seems even was able to create folders in profiles for unprivileged users; for examples for user 'gaio' (a privileged one) i can see `C:\Users\gaio\AppData\Local\AceCentre\FaceCommander` folder and within the `log.txt` logfile.

For the unprivileged user 'ospite' there's even no `C:\Users\ospite\AppData\Local\AceCenter` folder.

(so, indeed, i've no log to share)

**Comment by willwade** [comment]
Maybe stupid question - but have you got a camera connected to the pc? USB or other.. I have a feeling this crash is more complicated than user permissions.

**Comment by marcogaio** [comment]
Yes; i've still installed 'Project Gameface', and for the same unprivileged user Face Commander does not work while Project Gameface yes...

**Comment by willwade** [comment]
Ok. I think I see the problem. I can’t say I know how to fix this yet. I’m on holiday for the next week or so so give a little while..
